mobile/src/main/res/values/strings.xml

```diff
@@ -79,6 +79,7 @@
     <string name="error_http_code_495">Invalid SSL client certificate (HTTP response code 495)</string>
     <string name="error_http_code_496">SSL client certificate is required (HTTP response code 496)</string>
     <string name="error_http_code_502">Your reverse proxy got an invalid answer from openHAB (HTTP response code 502)</string>
+    <string name="error_http_code_503">openHAB is currently unavailable (HTTP response code 503)</string>
```

Comment:
Does the user understand, that this is only temporary?

Reply:
I think so, but what do you think about:
openHAB is temporary unavailable (HTTP response code 503)
?

Reply:
*temporarily ;-)